packages/store/package.json

```diff
@@ -16,23 +16,21 @@
   },
   "types": "ts/index.ts",
   "scripts": {
-    "build": "pnpm run build:tightcoder && pnpm run build:codegen && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
-    "build:abi": "forge build --out abi --skip test script",
-    "build:codegen": "tsx ./ts/scripts/tablegen.ts",
+    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
+    "build:abi": "forge build --extra-output-files abi --out abi --skip test script",
     "build:js": "vite build",
-    "build:tightcoder": "tsx ./ts/scripts/codegenTightcoder.ts && prettier --write '**/*.sol'",
+    "build:mud": "tsx ./ts/scripts/tablegen.ts",
     "build:typechain": "typechain --target ethers-v5 'abi/**/*.sol/!(*.abi).json'",
     "clean": "pnpm run clean:abi && pnpm run clean:typechain && pnpm run clean:js",
     "clean:abi": "rimraf abi",
     "clean:js": "rimraf dist",
     "clean:typechain": "rimraf types",
     "docs": "rimraf API && hardhat docgen",
     "gas-report": "../cli/dist/mud.js gas-report --path test/* --path test/**/* --save gas-report.json",
+    "generate-tightcoder": "tsx ./scripts/generate-tightcoder.ts && prettier --write '**/tightcoder/*.sol'",
```

Comment:
This looked more like a one-off script so I moved it out of the build path (doesn't need to be regenerated each time and is expected to be run locally and changes committed).

Reply:
this doesn't really change/matter, but if we have some localgen script for `generate-test-tables`, tightcoder could also use it